tasks/tekton-worker-image.yaml

```diff
@@ -0,0 +1,31 @@
+---
+apiVersion: image.openshift.io/v1
+kind: ImageStream
+metadata:
+  name: tekton-worker
+spec:
+  lookupPolicy:
+    local: false
+status:
+  dockerImageRepository: image-registry.openshift-image-registry.svc:5000/default/tekton-worker
+---
+apiVersion: build.openshift.io/v1
+kind: Build
```

Comment:
This should be a buildconfig imo, so that we could start a build on every pipeline run (it doesn't take long to be rebuilt every release)

Reply:
I agree. Since I'll make this a BuildConfig, should it live here or should it go in the `buildconfigs` dir?

Reply:
Yup